web-common/src/features/dashboards/selectors.ts

```diff
@@ -139,8 +142,13 @@ export const useModelHasTimeSeries = (
 export function useModelAllTimeRange(
```

Comment:
Since this is a custom hook we can just pass the enabled in here than repeating in all components